### PR TITLE
Update OCP cluster request for openstack-operator tempest job

### PR DESCRIPTION
**Pull request by viroel**
Testing with 4.13 first, before setting to 4.12 which is the current version

### CODE REVIEW

File: ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-main.yaml [hunk 1]
```diff
@@ -104,7 +104,7 @@ tests:
     owner: openstack-k8s-operators
     product: ocp
     timeout: 1h20m0s
-    version: "4.10"
```
Comment (viroel) [inline ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-main.yaml:107]: For some reason we missed this test during migration from 4.10 to 4.12.
It should be the last one:
`openstack-k8s-operators]$ grep "4.10" -r . ` returns nothing

**Comment by viroel** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-tempest

**Comment by viroel** [comment]
/hold

we don't want to merge this now

**Comment by viroel** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-tempest

**Comment by viroel** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-tempest

**Comment by viroel** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-tempest

**Comment by Sandeepyadav93** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-tempest

**Pull request closed without merging**